docs/static_site/src/pages/get_started/build_from_source.md

```diff
@@ -36,6 +35,7 @@ Building from source follows this general two-step flow of building the shared l
 1. Build the MXNet shared library, `libmxnet.so`.
     * [Clone the repository](#clone-the-mxnet-project)
     * [Prerequisites](#prerequisites)
+        * [Compiler requirement](#compiler-requirement)
```

Comment:
looks like the math library selection part is broken, can you take a look and fix it in this PR?

Reply:
It's fixed now. Thanks for pointing it out!